pages/windows/azcopy.md

```diff
@@ -9,19 +9,19 @@
 
 - Upload a local file:
 
-`azcopy copy '{{path/to/source/file}}' 'https://{{storage-account-name}}.blob.core.windows.net/{{container-name}}/{{blob-name}}'`
+`azcopy copy '{{path/to/source/file}}' 'https://{{storage_account_name}}.blob.core.windows.net/{{container_name}}/{{blob_name}}'`
```

Comment:
Maybe this was made because `_` is not valid in URLs. But we don't need that the example URL works :man_shrugging:

Reply:
It isn't? I didn't know that. In that case, we should revert this one.

Reply:
Underscores in URLs are valid, e.g., https://en.wikipedia.org/wiki/Main_Page. I don't see any special reason not to use them :)

Reply:
I mean for the subdomain, you can't use underscore (`_`) but you can use hyphens (`-`)

Reply:
You're right, but then only the first token would require the hyphen. Of course, it'd be very ugly to use hyphens and underscores in the same example and we should stick to one :) IMHO: validity shouldn't be an issue with tokens

Reply:
Generally we prefer underscores in token syntax elsewhere, so for consistency I think underscores might be the better choice here.